E/Epsteinlib/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Epsteinlib"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/epsteinlib/epsteinlib.git", "2a367d46d5f6a843858fc1b48beee9ce8eb4483d")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+pip install cython
+cd $WORKSPACE/srcdir/epsteinlib
+meson setup build --cross-file=${MESON_TARGET_TOOLCHAIN} --buildtype=release -Dbuild_python=false
+ninja -C build -j${nproc}
+ninja -C build install
+"""
```

Comment:
Need to install the license: https://buildkite.com/julialang/yggdrasil/builds/24222#0199f45c-f5a3-45f2-8547-a794f2c43581/1098-1287
```
┌ Error: Unable to find valid license file in "${prefix}/share/licenses/Epsteinlib"
```
```suggestion
install_license LICENSES/*
"""
```